bitnami/concourse/values.yaml

```diff
@@ -0,0 +1,1144 @@
+      port: http
+    failureThreshold: 1
+    successThreshold: 1
+
+  ## @param web.customLivenessProbe Custom livenessProbe that overrides the default one
+  ##
+  customLivenessProbe: {}
+  ## @param web.customReadinessProbe Custom readinessProbe that overrides the default one
+  ##
+  customReadinessProbe: {}
+  ## web resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param web.resources.limits The resources limits for the web containers
+  ## The requested resources for the web containers
+  ## @param web.resources.requests The requested for the web containers
+  ## resources:
+  ##   limits: {}
+  ##   requests: 
+  ##     cpu: "100m"
+  ##     memory: "128Mi"
+  resources:
+    limits: {}
+    requests: {}
+  
+  ## @param web.tls.enabled enable serving HTTPS traffic directly through the web component.
+  ## @param web.tls.containerPort on which to listen for HTTPS traffic.
+  tls:
+    enabled: false
+    containerPort: 443
+
+  ## @param web.existingConfigmap The name of an existing ConfigMap with your custom configuration for web
+  ##
+  existingConfigmap:
+  ## @param web.command Override default container command (useful when using custom images)
+  ##
+  command: []
+  ## @param web.hostAliases Deployment pod host aliases
+  ##
+  hostAliases: []
+  ## @param web.podLabels Extra labels for web pods
+  ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+  ##
+  podLabels: {}
+  ## Where secrets should be mounted for the web container.
+  ##
+  ## @param web.keySecretsPath 
+  ## @param web.teamSecretsPath
+  keySecretsPath: "/concourse-keys"
+  teamSecretsPath: "/team-authorized-keys"
+
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param web.podSecurityContext.enabled Enabled web pods' Security Context
+  ## @param web.podSecurityContext.fsGroup Set web pod's Security Context fsGroup
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 1001
+  ## Configure Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param web.containerSecurityContext.enabled Enabled web containers' Security Context
+  ## @param web.containerSecurityContext.runAsUser Set web containers' Security Context runAsUser
+  ##
+  containerSecurityContext:
+    enabled: true
+    runAsUser: 1001
+  
+  ## @param web.podSecurityPolicy.create Specifies whether a PodSecurityPolicy should be created (set `podSecurityPolicy.create` to `true` to enable it)
+  podSecurityPolicy:
+    create: false
+
+  service: 
+    ## @param web.service.type For minAvailableikube, set this to ClusterIP, elsewhere use LoadBalancer or NodePort
+    ## Ref: https://kubernetes.io/docs/user-guide/services/#publishing-services---service-types
+    ##
+    type: ClusterIP
+
```

Comment:
missing the `port` section, as one thing is the service port and another the `containerPort`